lib/app/utils.js

```diff
@@ -132,11 +132,23 @@ export async function setContext(app, context) {
         path = status
         status = 302
       }
-      app.context.next({
-        path: path,
-        query: query,
-        status: status
-      })
+      if (path.charAt(0) === '/' && path.charAt(1) !== '/') {
+        app.context.next({
+          path: path,
+          query: query,
+          status: status
+        })
+      } else {
+        path = formatUrl(path, query)
+        if (process.server) {
+          app.context.res.setHeader('Location', path)
+          app.context.res.statusCode = status
+          app.nuxt._redirected = true
+        }
+        if (process.client) {
+          window.location = path
```

Comment:
This should be `window.location.replace(path)`. The current form adds an additional history item.

Reproduce by clicking a redirect link, then hit Back twice. It should go back after one click.

Reply:
Already fixed in dev branch :)

Reply:
Thank you @ydnar

Reply:
Sweet as!